R/available_forecasts.R

```diff
@@ -48,13 +49,13 @@ get_forecast_counts <- function(data,
     by <- forecast_unit
   }
 
-  # collapse several rows to 1, e.g. treat a set of 10 quantiles as one,
+  # collapse several rows to 1, e.g. treat a set of 10 s as one,
```

Comment:
is this intentional or do you mean. 

```suggestion
  # collapse several rows to 1, e.g. treat a set of 10 quantile levels as one,
```

Reply:
```suggestion
  # collapse several rows to 1, e.g. treat a set of 10 quantiles as one,
```

Ah good catch, thanks! this is what I meant. I first had quantile levels and then reverted it because I think you're really collapsing a set of quantiles and treating them as one forecast (because the forecast is made of several quantiles, not of several quantile levels).